icu4j/maven-migration/pom.xml

```diff
@@ -39,34 +39,40 @@
     <proj-title>International Components for Unicode for Java</proj-title>
     <proj.displayname>${project.artifactId}</proj.displayname>
 
-    <mf.Manifest-Version>1.0</mf.Manifest-Version>
-    <mf.Specification-Title>${proj-title} (${proj.displayname})</mf.Specification-Title>
-    <mf.Implementation-Title>${proj-title} (${proj.displayname})</mf.Implementation-Title>
-    <mf.Bundle-Description>${proj-title} (${proj.displayname})</mf.Bundle-Description>
-    <mf.Automatic-Module-Name>com.ibm.icu.${project.artifactId}</mf.Automatic-Module-Name>
-
-    <api.doc.version>73 Release Candidate</api.doc.version>
     <!-- Variables used to generate the javadoc -->
+    <api.doc.version>74 Release Candidate</api.doc.version>
     <icu4j.api.doc.window.title>ICU4J ${api.doc.version}</icu4j.api.doc.window.title>
     <icu4j.api.doc.header>ICU4J ${api.doc.version}</icu4j.api.doc.header>
     <icu4j.api.doc.copyright.footer>&lt;font size=-1&gt;Copyright © 2016 Unicode, Inc. and others.&lt;/font&gt;</icu4j.api.doc.copyright.footer>
     <!-- Needs to be overriden in sub-modules -->
     <icu4j.api.doc.root.dir>${project.basedir}</icu4j.api.doc.root.dir>
 
-    <mf.Specification-Version>73</mf.Specification-Version>
-    <mf.Implementation-Version>73.0.1</mf.Implementation-Version>
-    <mf.Bundle-Version>73.0.1</mf.Bundle-Version>
+    <!-- Used for the .jar Manifest files -->
```

Comment:
Sorry, this looks ugly, but it is mostly reordering to group manifest properties together.
But better now than after the migration, when it can annoy more people :-)